File: PRAK304-2210817210023-Muhammad Shofy Akmal.py
a = float(input("Masukkan Bilangan Cacah : "))
if a == 0:
    print ("Nol")
elif 0<a<10 :
    print ("Satuan")
elif 10<a<20 :
    print ("Belasan")
elif  a>=100 :
    print ("Anda Menginput Melebihi Limit Bilangan") 
else :
    print ("Puluhan")

File: PRAK304-2210817210023-MuhammadShofyAkmal.c
#include <stdio.h>
int main () {
int a;
printf ("Masukkan Bilangan Cacah : "); scanf ("%d", &a);
if      (a == 0)  
{printf ("Nol");                                   }
else if (a>0 & a<10)  
{printf ("Satuan");                                }
else if (a>10 & a<20)  
{printf ("Belasan");                               }
else if (a>=100 )  
{printf ("Anda Menginput Melebihi Limit Bilangan");}
else               
{printf ("Puluhan");                               }
}